README.md

```diff
@@ -182,6 +182,14 @@ First off, you need to `--init` terraform on your local machine, by executing th
     If `--send-state-sync` hasn't been used before, only checkpoints will be detected. The execution stops when
     a `state-sync` is found
 
+- ` ../../bin/express-cli --instances-stop`
+
+  - Stop the AWS EC2 VM instances associated with the deployed devnet.
+
+- ` ../../bin/express-cli --instances-start`
+
+  - Start the (previously stopped) AWS EC2 VM instances associated with the deployed devnet. Also, it starts all services, such as ganache, heimdall, and bor
```

Comment:
If I have more than one cluster, is there a way to tell the CLI which cluster I would like to start or stop?

Reply:
Yes, this is possible since we implemented the multi-devnets approach.
When you `init` a new cluster, a folder gets created under `./deployments/devnet-<ID>` and it contains the `terraform` state and the `.env`.
Executing these commands in the proper devnet folder will only start/stop that specific cluster.